packages/node_modules/@node-red/util/lib/log.js

```diff
@@ -75,28 +87,27 @@ LogHandler.prototype.shouldReportMessage = function(msglevel) {
            msglevel <= this.logLevel;
 }
 
-
 // Older versions of Node-RED used the deprecated util.log function.
 // With Node.js 22, use of that function causes warnings. So here we
 // are replicating the same format output to ensure we don't break any
 // log parsing that happens in the real world.
 const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
-const utilLog = function (msg) {
+const utilLog = function (msg, level) {
     const d = new Date();
     const time = [
         d.getHours().toString().padStart(2, '0'),
         d.getMinutes().toString().padStart(2, '0'),
         d.getSeconds().toString().padStart(2, '0')
     ].join(':');
-    console.log(`${d.getDate()} ${months[d.getMonth()]} ${time} - ${msg}`)
+    console.log(chalk[levelColours[level]](`${d.getDate()} ${months[d.getMonth()]} ${time} - ${msg}`))
```

Comment:
We should guard against `levelColours[level]` not being defined and apply a sensible default.
```suggestion
            console.log(chalk[levelColours[level] || 'white'](`${d.getDate()} ${months[d.getMonth()]} ${time} - ${msg}`))
```